monitoring/.codegen.yaml

```diff
@@ -1,3 +1,5 @@
 schemapatch:
   requiredFeatureSets:
+  - ""
+  - "Default"
   - "TechPreviewNoUpgrade"
```

Comment:
This produces
```
error running generator swaggerdocs on group monitoring.openshift.io:
	error generating swagger docs for monitoring.openshift.io/v1alpha1:
		found 4 missing comments:
In struct: AlertRelabelConfig, field documentation is missing: metadata
In struct: AlertRelabelConfigList, field documentation is missing: metadata
In struct: AlertingRule, field documentation is missing: metadata
In struct: AlertingRuleList, field documentation is missing: metadata
```
Same for `v1` of course. Seems like other APIs have the same "issue"?

Reply:
Yes, you're right, I was expecting this, the fix for each of those warnings if to find the `metadata` field and add a comment, which you can copy/paste from here onto each one in both APIs to fix
```
	// metadata is the standard object's metadata.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
> Seems like other APIs have the same "issue"?

They do indeed, not only on metadata, but in a wider scope too, my plan is to fix these up in time, so that no warnings are produced

Reply:
Ack! I checked the example api, but had no luck.